utils/swift_build_support/swift_build_support/products/wasmswiftsdk.py

```diff
@@ -182,6 +212,7 @@ def _build_target_package(self, swift_host_triple, has_pthread,
                            env={'DESTDIR': clang_dir})
 
         self._build_libxml2(swift_host_triple, has_pthread, wasi_sysroot)
+        self._build_foundation(swift_host_triple, has_pthread, wasi_sysroot)
```

Comment:
Is Foundation built for any other platform for `smoke test` CI trigger? If not, I think we should make this conditional on `--foundation` passed to `build-script` or included in a preset